### DISCUSSION TITLE
Feature: Cascading Settings

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#2317] [Show and tell]
This thread is to announce the launch of a major new feature: cascading settings. Starting today, you can set defaults for Pub settings at the Community and Collection levels, and have them "cascade" down to Pubs. You can also, of course, override them at the Pub level.

Our hope is this saves community admins a ton of configuration time, reduces production errors, and allows folks using our submission feature to pre-set default settings based on collections to streamline their workflows.

It's now live, and we have initial documentation for the feature here: https://help.pubpub.org/pub/2c3n4f5g

Feel free to leave feedback, thoughts on iterations you'd like to see, any bug reports (though we've done a lot of testing and hope these are minimal) in this thread.

Finally, a huge thanks to @idreyn, who's responsible for much of this new feature, from conception to launch to documentation.

**Comment by jeffpooley** [comment]
I want to chime in with thanks to @idreyn ! This is a huge update.

**Comment by kyledenlinger** [comment]
Christmas come early! This is amazing! Thanks @idreyn!

**Comment by tosteiner** [comment]
Wow, this really looks amazing - thank you, @idreyn for making this possible!  🎉

**Comment by metasj** [comment]
A gift!  Now if only one of the cascading settings were "drafts default to public" -- where any visitor can see that drafts exist and go to the latest draft...

**Comment by kyledenlinger** [comment]
Working with cascading styles quite extensively at the moment. I would really appreciate it if pub preview images could also cascade, or simply default to the pub header background image if one is set at the collection level

**Comment by amdeis** [comment]
Is there a requirement on what KINDS of "collections" get cascading pub settings? For example, in HDSR issue collections have a "Pub settings" tab in the collection settings (https://hdsr.mitpress.mit.edu/dash/collection/volume5issue3/settings/pub-settings), but HDSR's "Editors" collection doesn't (https://hdsr.mitpress.mit.edu/dash/collection/editors/settings/details)
